packages/Parallel/ValueObject/Bridge.php

```diff
@@ -24,11 +24,6 @@ final class Bridge
      */
     public const SYSTEM_ERRORS_COUNT = 'system_errors_count';
 
-    /**
-     * @var string
-     */
-    public const FILES = 'files';
```

Comment:
is this a BC break? I don't think so as this enum looks internal, and no `@æpi` annotation